src/many/src/server/module/_1000_abci_backend.rs

```diff
@@ -51,6 +52,67 @@ pub struct AbciCommitInfo {
     pub hash: ByteVec,
 }
 
+#[derive(Encode, Decode, Debug)]
+#[cbor(map)]
+pub struct Snapshot {
+    #[n(0)]
+    pub path: PathBuf,
+    #[n(1)]
+    pub height: u64,
+    #[n(2)]
+    pub hash: Vec<u8>,
+ //   #[n(3)]
+ //   pub chunks: u64,
```

Comment:
?

Reply:
Added chunks back to reflect TM Snapshot struct